src/vsphere_cpi/spec/integration/datastore_cluster_spec.rb

```diff
@@ -187,7 +187,7 @@
         [],
         {}
       )
-      end.to raise_error(/No valid placement found for disks/)
+      end.to raise_error(/No valid placement found/)
```

Comment:
Why're we looking for a different error message than on line 59?

Reply:
I think I copied at different times. Considering that it is a regex , I did not catch it. Correcting it.